src/Controls/src/Core/PlatformConfiguration/WindowsSpecific/Page.cs

```diff
@@ -9,35 +9,56 @@ namespace Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific
 {
 	using FormsElement = Maui.Controls.Page;
 
-	/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="Type[@FullName='Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific.Page']/Docs/*" />
+	/// <summary>
+	/// The page instance that Microsoft.Maui.Controls created on the Windows platform.
+	/// </summary>
 	public static class Page
 	{
 		#region ToolbarPlacement
 
-		/// <summary>Bindable property for <see cref="ToolbarPlacement"/>.</summary>
+		/// <summary>
+		/// Backing store for the attached property that controls the placement of the toolbar.
+		/// </summary>
 		public static readonly BindableProperty ToolbarPlacementProperty =
 			BindableProperty.CreateAttached("ToolbarPlacement", typeof(ToolbarPlacement),
 				typeof(FormsElement), ToolbarPlacement.Default);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='GetToolbarPlacement'][1]/Docs/*" />
+		/// <summary>
+		/// Returns a value that controls the placement of the toolbar.
+		/// </summary>
+		/// <param name="element">The platform specific element on which to perform the operation.</param>
+		/// <returns>A value that controls the placement of the toolbar.</returns>
 		public static ToolbarPlacement GetToolbarPlacement(BindableObject element)
 		{
 			return (ToolbarPlacement)element.GetValue(ToolbarPlacementProperty);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='SetToolbarPlacement'][1]/Docs/*" />
+		/// <summary>
+		/// Sets a value that controls the placement of the toolbar.
```

Comment:
This doesn't actually do anything right now. Is that useful to note ?

Reply:
@jsuarezruiz

Reply:
https://github.com/dotnet/maui/blob/7d21950c98f2f7d52f2a65b006ca2128e6204d9e/src/Controls/src/Core/Toolbar/Toolbar.Windows.cs#L170
Not implemented

Reply:
I think there are 2 approaches here: 

1. We can make it invisible from the IDE by addding `[EditorBrowsable(EditorBrowsableState.Never)]`
2. Adding a remark saying that this functionality hasn't been implemeted yet.

The ideal scenario would be implementing the missing functionality, but that may go beyond the scop of this PR. 

@davidbritch what would be your recommended approach here?

Reply:
If the API does nothing right now I'd hide it in the IDE.

Reply:
@jsuarezruiz. With this change I think we should be good to merge. I'd hate for this PR to go stale